test/distributed/org/apache/cassandra/distributed/test/sai/SAIUtil.java

```diff
@@ -86,23 +86,26 @@ public static void assertIndexQueryable(Cluster cluster, String keyspace, String
      */
     private static void assertIndexesQueryable(Cluster cluster, String keyspace, final Iterable<String> indexes)
     {
-        IInvokableInstance localNode = cluster.get(1);
         final List<InetAddressAndPort> nodes =
             cluster.stream()
                    .map(node -> nodeAddress(node.broadcastAddress()))
                    .collect(Collectors.toList());
 
-        localNode.runOnInstance(() -> {
-            for (String index : indexes)
-            {
-                for (InetAddressAndPort node : nodes)
+        for (var localNode : cluster)
```

Comment:
The only thing we could do to make this a tiny bit more efficient is remember which nodes have already seen all the other nodes' indexes in a queryable state, but with a delay of 60s in the first place, the time we save doing that might not matter.